widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadView.kt

```diff
@@ -208,9 +221,14 @@ class ChatHeadView @JvmOverloads constructor(
         })
     }
 
-    private fun updateOperatorPlaceholderImageView() {
+    private fun updatePlaceholderImageView() {
+        val placeholderIcon = if (isCallVisualizerUseCase.execute()) {
+            R.drawable.ic_screensharing
```

Comment:
Can you please add here `TODO: MOB-1942` so that I don't forget to add this icon to UiTheme the same way as `operatorPlaceholderIcon`

Reply:
Added ✅